### PR TITLE
Add sphinx-releases

### PR DESCRIPTION
**Pull request by kwilcox**

### CODE REVIEW

File: recipes/sphinx-releases/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "sphinx-releases" %}
```
Comment (jakirkham) [inline recipes/sphinx-releases/meta.yaml]: Can understand the value of disambiguation here. Though `releases` is the PyPI name. Thoughts?

cc @conda-forge/staged-recipes

**Comment by jakirkham** [comment]
Other than the naming question, LGTM.

File: recipes/sphinx-releases/meta.yaml (hunk 1 above)
Comment (kwilcox) [reply recipes/sphinx-releases/meta.yaml]: `releases` is an unfortunate name. Other conda sphinx plugins use the `sphinx-` prefix so I thought it best to follow suit. I don't have strong opinions either way.

**Comment by dougalsutherland** [comment]
I'd be okay with either `releases` or `sphinx-releases`, personally. (`releases` is pretty generic, but for the record, there's not anything else named that on either Homebrew or Ubuntu repos; Ubuntu packages it as `python-releases`.)

Also LGTM other than making a decision there. @jakirkham, if you don't feel strongly one way or another I'd be happy to merge like this.

**Comment by scopatz** [comment]
I am personally in favor of the name `sphinx-releases`

**Comment by jakirkham** [comment]
Honestly was torn. Hence why was secretly hoping others would weigh in and make the decision. :)

**Comment by sodre** [comment]
@conda-forge/staged-recipes, last chance to request a name change on `sphinx-release`... If I don't hear anything back I will merge it tomorrow night.

**Comment by jakirkham** [comment]
FYI we are having conversion problems as noted in issue ( https://github.com/conda-forge/staged-recipes/issues/5846 ).

**Pull request merged**